docs/source/change_log.rst

```diff
@@ -15,6 +15,8 @@ Version 1.14.0
 
 * Added the ``Message-ID`` MIME header to outgoing messages
 * Attempt SSH authentication with all agent-provided SSH keys
+* Deleted `Pipefile.lock` from repository to prevent hash issues between python interpreter versions.
```

Comment:
So this isn't markdown, to mark something as a literal it should be two back ticks like Message-ID before. Also Python should be capitalized and the local convention here is to not use periods.

Same comments for the line below.